src/main/java/org/apache/commons/lang3/builder/DiffView.java

```diff
@@ -0,0 +1,64 @@
+package org.apache.commons.lang3.builder;
+
+/**
+ * <p>A {@code DiffView} object holds the string representation of the difference of
+ * a given field between two objects.</p>
+ *
+ * @since 3.10
+ */
+public class DiffView {
+	private final String field;
+	private final String leftValue;
+	private final String rightValue;
+
+	/**
+	 * <p>Constructs a {@code DiffView} from 3 string params: field, leftValue and rightValue</p>
+	 *
+	 * @param field designates the field identified as different.
+	 * @param leftValue holds the string representation of the value in the left side of the comparison.
+	 * @param rightValue holds the string representation of the value in the right side of the comparison.
+	 */
+	public DiffView(String field, String leftValue, String rightValue) {
+		this.field = field;
```

Comment:
I think the files were formatted with tabs, and this may cause issues in Travis when it builds using Checkstyle.